eng/testing/tests.mobile.targets

```diff
@@ -14,7 +14,9 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(TargetOS)' == 'Android'">
-    <RunScriptCommand>$HARNESS_RUNNER android test --instrumentation="net.dot.MonoRunner" --package-name="net.dot.$(AssemblyName)" --app=$EXECUTION_DIR/bin/$(TestProjectName).apk --output-directory=$XHARNESS_OUT -v --</RunScriptCommand>
+    <_expectedExitCode>0</_expectedExitCode>
+    <_expectedExitCode Condition="'$(TestProjectName)' == 'HelloAndroid'">42</_expectedExitCode>
```

Comment:
we can move this to the sample .csproj, e.g. set `<ExpectedExitCode>42</ExpectedExitCode>` there and then check whether it is set here with:

```xml
 <ExpectedExitCode Condition="'$(ExpectedExitCode)' == ''">0</ExpectedExitCode>
```

This works because at the end of the day the sample.csproj imports tests.mobile.targets so we can use values in the .csproj to override behavior here.

Reply:
Perhaps, this change is not relevant at the moment.

Reply:
Reverted this change for now